pkgs/top-level/python-packages.nix

```diff
@@ -1860,14 +1862,18 @@ in {
     inherit (pkgs.mxnet) name version src meta;
 
     buildInputs = [ pkgs.mxnet ];
-    propagatedBuildInputs = with self; [ requests numpy graphviz ];
+    propagatedBuildInputs = with self; [ requests numpy graphviz onnx ];
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Done